### PR TITLE
Update database by adding mediaContributors support

### PR DESCRIPTION
**Pull request by jaisal1024**
Added the table mediaContributors_post with author_id's and post_id's to support a many-to-many relationship between authors (who are media contributors) and posts (articles). This will be added to the knex database in the script.

### CODE REVIEW

File: updateDatabaseMediaContributors [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const MAX_NAME_OR_SLUG_LENGTH = 70;
+const MAX_DESCRIPTION_LENGTH = 170;
+const MAX_BIOGRAPHY_LENGTH = 255;
+
+
+// Start of script - 1. Add mediaContributors_posts table with reference to author_id in row 1 and posts_id in row 2
+knex.schema
+//Reset Database
+.dropTableIfExists('mediaContributors_posts')
```
Comment (emilgoldsmith) [inline updateDatabaseMediaContributors]: Maybe just for consistent naming conventions make it `media_contributors_posts` ?

**Review by emilgoldsmith — CHANGES_REQUESTED** [review]
If you just change the name I think everything looks good :). This is exactly the same as authors_posts right? (that's meant as a good thing)

**Comment by jaisal1024** [comment]
I just made the requested changes. And yes it is basically the exact same as the other script. But obviously, you should double check before we finalize the code.

**Review by emilgoldsmith — APPROVED** [review]
Yep looks good now :)

**Pull request merged**